lib/travis/listener.rb

```diff
@@ -15,7 +15,7 @@ def config
 
   module Listener
     class Config < Travis::Config
-      define  redis:   { url: 'redis://localhost:6379', namespace: 'sidekiq', network_timeout: 5 },
+      define  redis:   { url: ENV['LISTENER_REDIS_URL'] || 'redis://localhost:6379', namespace: 'sidekiq', network_timeout: 5 },
```

Comment:
My preference for `LISTENER_REDIS_URL` is to have consistent naming across all apps, and not have them be context specific. I like the idea of being able to look at the code and instantly tell which redis instance is being referred to.

I am open to changing it to `REDIS_URL` though. Let me know what you think.